public/app/plugins/datasource/cloudwatch/variables.ts

```diff
@@ -162,4 +132,21 @@ export class CloudWatchVariableSupport extends CustomVariableSupport<CloudWatchD
       expandable: true,
     }));
   }
+
+  allMetricFindValue: MetricFindValue = { text: 'All', value: allOption.value, expandable: true };
+  async handleAccountsQuery({ region }: VariableQuery) {
+    return this.api.getAccounts({ region }).then((accounts) => {
+      const metricFindOptions = accounts.map((account) => ({
+        text: account.label,
+        value: account.id,
+        expandable: true,
+      }));
+
+      return metricFindOptions.length ? [this.allMetricFindValue, ...metricFindOptions] : [];
+    });
+  }
+}
```

Comment:
i'm not sure what the best way to ask is, but i'll try here:

if i'm reading this correctly, the CloudWatch label is displayed in the dashboard query variables preview section (e.g., dev, beta, prod) but the regular expression is applied to the value. Is this the expected behavior and is it documented anywhere?